Makefile

```diff
@@ -22,7 +22,8 @@ regen_check: $(GIR) $(GIR_FILES)
 src/auto/mod.rs : Gir.toml $(GIR) $(GIR_FILES)
 	$(GIR) -c Gir.toml
 
-gir-sys : sourceview-sys/src/lib.rs
+gir-sys: sourceview-sys/src/lib.rs
+	cd sourceview-sys && cargo fmt
```

Comment:
IMHO better move `cd sourceview-sys && cargo fmt` from `gir : src/auto/mod.rs`
to `sourceview-sys/src/lib.rs`.
And maybe `cargo fmt` from `gir : src/auto/mod.rs` to `src/auto/mod.rs`

Reply:
Or just change `gir : src/auto/mod.rs` to `gir : sourceview-sys/src/lib.rs src/auto/mod.rs`, not sure which is better

Reply:
Hm, seems I wrong again, your version do my first suggestion,  but `cd sourceview-sys && cargo fmt` just was unneeded in `gir:` part.

Reply:
I moved it into the rule. Tell me if my change sounds good to you.

Reply:
Thanks you, sorry for nagging

Reply:
No you were right, It allowed us to clean some Makefile, which is always a good thing. :)